tests/utils.py

```diff
@@ -43,7 +42,7 @@
 -----END PRIVATE KEY-----"""
 
 
-def create_rohmu_config(tmpdir: py.path.local, *, compression: bool = True, encryption: bool = True) -> RohmuConfig:
+def create_rohmu_config(tmpdir: os.PathLike, *, compression: bool = True, encryption: bool = True) -> RohmuConfig:
```

Comment:
tmpdir (the python fixture) is really a py.path.local, which has a rather different API from Path.

However, it looks like we're actually passing a path from `tmp_path` (the fixture), which is really a Path. I think it would be clearer to rename all the `tmpdir` to `tmp_path` and type everything as `Path`.

Reply:
Didn't want to do the refactoring in this PR, but I can.

Reply:
It's not a refactoring just a rename, done.